src/Autofac/ContainerBuilder.cs

```diff
@@ -139,11 +140,11 @@ public ContainerBuilder RegisterBuildCallback(Action<IContainer> buildCallback)
         /// first create the container, then call Update() on the builder.
         /// </remarks>
         /// <returns>A new container with the configured component registrations.</returns>
-        public IContainer Build(ContainerBuildOptions options = ContainerBuildOptions.None)
+        public IContainer Build(ContainerBuildOptions options = ContainerBuildOptions.None, DefaultServiceFlags flags = DefaultServiceFlags.All)
```

Comment:
This is where a second overload to `Build` might prevent breaking changes by mapping `ExcludeDefaultModules` to `DefaultServiceFlags.None`.

Reply:
It appears to me that the `DefaultServiceFlags` (or `FeatureToggles` or `ContainerFeatures` or whatever) is an attempt at a more granular way to include or exclude default features... in which case, yeah, an overload where the current `ContainerBuildOptions` maps to some set of features would be better.